### PR TITLE
Make Travis run ok

### PR DESCRIPTION
**Pull request by bcgianni**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -3,9 +3,7 @@
 
 require 'rspec'
 require 'agendor-ruby'
-require 'codeclimate-test-reporter'
```
Comment (pedrovitti) [inline spec/spec_helper.rb]: :(

Comment (bcgianni) [inline spec/spec_helper.rb]: this method of reporting to CC has been deprecated :(

https://github.com/codeclimate/ruby-test-reporter/blob/master/README.md

Comment (pedrovitti) [inline spec/spec_helper.rb]: great, I thought you were done with the PR. 😎

**Comment by bcgianni** [comment]
what about now @pedrovitti

**Comment by pedrovitti** [comment]
:shipit:

**Pull request merged**